lib/output/utils.js

```diff
@@ -48,12 +68,18 @@ function tryImplForWrapper(wrapper) {
 const iterInternalSymbol = Symbol("internal");
 const IteratorPrototype = Object.getPrototypeOf(Object.getPrototypeOf([][Symbol.iterator]()));
 
-module.exports.wrapperSymbol = wrapperSymbol;
-module.exports.implSymbol = implSymbol;
-module.exports.getSameObject = getSameObject;
-module.exports.wrapperForImpl = wrapperForImpl;
-module.exports.implForWrapper = implForWrapper;
-module.exports.tryWrapperForImpl = tryWrapperForImpl;
-module.exports.tryImplForWrapper = tryImplForWrapper;
-module.exports.iterInternalSymbol = iterInternalSymbol;
-module.exports.IteratorPrototype = IteratorPrototype;
+module.exports = exports = {
+  isObject,
+  getReferenceToBytes,
+  getCopyToBytes,
+  mixin,
+  wrapperSymbol,
+  implSymbol,
+  getSameObject,
+  wrapperForImpl,
+  implForWrapper,
+  tryWrapperForImpl,
+  tryImplForWrapper,
+  iterInternalSymbol,
+  IteratorPrototype
+};
```

Comment:
Please don't reformat existing files for no particular gains.

Reply:
@ExE-Boss you are not a reviewer for this project, so please do not impose your style advice (which often conflicts with the maintainers) on others. We've been mostly able to ignore your reviews so far but if you are rude, then we'll have to take larger measures.